index.html

```diff
@@ -17,43 +17,61 @@
 
 <body>
   <header>
-    <div class="logo">
-      <a href="" id="logo">Bright Ezeamaka</a>
+    <div class="logo"><a href="">Bright Ezeamaka</a></div>
+    <div class="dropdown">
+      <img src="./Images/Icon - Menu.png" alt="img">
+    </div>
+    <div class="deskstopNav">
+      <ul class="deskstop-ul">
+        <li> <a href="#" class="deskstop-li">Portoflio</a> </li>
+        <li> <a href="#" class="deskstop-li">About</a></li>
+        <li> <a href="#" class="deskstop-li">Contact </a></li>
+        <li class="deskstop-li"> <a href="#"></a><img src="./Images/Icon - Mail.png" alt="img"></li>
+      </ul>
     </div>
-    <nav class="dropdown">
-      <img src="./Images/Icon - Menu.png" alt="">
-    </nav>
   </header>
   <section class="headline">
     <div class="headline-container">
-      <h1>Hey There. I'm Bright</h1>
-      <p>
-        <span>I’am a Software Developer</span>
-      </p>
-      <p>
-        I can help you build a product , feature or website Look through some of my work and
-        experience! If you like what you see and have a project you need coded,
-        don’t hestiate to contact me.
-      </p>
-      <ul>
-        <li class="icon"> <a href=""><i class="fa-brands fa-github"></i></a></li>
-        <li> <a href=""><i class="fa-brands fa-linkedin"></i></a></li>
-        <li> <a href=""><i class="fa-brands fa-angellist"></i></a></li>
-        <li> <a href=""><i class="fa-brands fa-twitter"></i></a></li>
-        <li> <a href=""><i class="fa-solid fa-m"></i></a></li>
-
-      </ul>
+      <div class="headline-text">
+        <div class="headline-p">
+          <h1>Hey There. I'm Bright</h1>
+          <p>
+            <span>I’am a Software Developer</span>
+          </p>
+          <p class="text">
+            I can help you build a product , feature or website Look through some of my work and
+            experience! If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.
+          </p>
+        </div>
+        <div>
+          <ul class="headline-icon">
+            <li><i class="fa-brands headlineIcon fa-github"></i></li>
+            <li><i class="fa-brands headlineIcon fa-linkedin-in"></i></li>
+            <li><i class="fa-brands headlineIcon fa-angellist"></i></li>
+            <li><i class="fa-brands headlineIcon fa-twitter"></i></li>
+            <li><i class="fa-solid headlineIcon fa-m"></i></li>
+          </ul>
+        </div>
+      </div>
+      <div class="headline-image">
+        <img src="./Images/Illustration-header@2x.png" class="headline-img" alt="img">
+      </div>
```

Comment:
- [ ] As stated by the previous reviewer, kindly fix the design to match the figma template. according to the requirements, your desktop version should be responsive from the screen size of `786px`, but this is what yours look like below :

![image](https://user-images.githubusercontent.com/60083437/220931054-d5757746-5b7d-4b11-9972-37e55c80547e.png)


***Suggestion you can consider***
- To fix this issue, I think you can consider using `background-image` in `CSS`. You have used  `img` tag in `HTML` file for the background image, but it is not the right way to set a background image.

>       <div class="headline-image"> <img src="./Images/Illustration-header@2x.png" class="headline-img" alt="img"> </div>  -> This is the wrong way.

- Rather use `background-image: url();`, like the same way you did for the other classes, in your [CSS](https://github.com/brightbrain20/my-portfolio/blob/desktop-version/styles.css). Here:

![image](https://user-images.githubusercontent.com/60083437/220932935-8b34cb85-5aec-4bfb-85fc-3bdacc07ff43.png)

Happy Coding 😄